src/main/webapp/destinationCreation.html

```diff
@@ -3,11 +3,18 @@
   <head>
     <meta charset="UTF-8"/>
     <title>Destination Creation</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
```

Comment:
What is this used for?

Reply:
This gives me access to a magnifying glass icon for the search button.